plugin/api.go

```diff
@@ -334,11 +334,16 @@ type API interface {
 	// Minimum server version: 5.10
 	SearchPostsInTeam(teamId string, paramsList []*model.SearchParams) ([]*model.Post, *model.AppError)
 
-	// AddChannelMember creates a channel membership for a user.
+	// AddChannelMember joins a user to a channel (as if they joined themselves)
```

Comment:
Suggest we add a bit of documentation here (and below) about the notification the user won't/will receive